x-pack/legacy/plugins/canvas/canvas_plugin_src/strings/apply_strings.ts

```diff
@@ -32,8 +34,41 @@ export const applyElementStrings = (elements: ElementFactory[]) => {
       if (displayName) {
         result.displayName = displayName;
       }
+
+      // Set translated tags
+      if (result.tags) {
+        result.tags = result.tags.map(tag => {
+          if (tag in TagStrings) {
```

Comment:
nit: I try to avoid `property in object` calls, because it also applies to the prototype chain.  YMMV

Reply:
Yeah.. when I looked through this again I ripped it out. Probably harmless but better safe than sorry